requirements.txt

```diff
@@ -1,2 +1,3 @@
 streamlit
-bump-my-version
\ No newline at end of file
+bump-my-version
+streamlit_js_eval
```

Comment:
Why do we need this library?

Reply:
It had to prepare another component because components couldn't pass the values obtained from JavaScript to the component self.

Reply:
It's possible to achieve this without using external libraries. However, as I said above that would require adding another separate component within streamlit_pdf_viewer to handle receiving results from JavaScript and to send data from JavaScript.
This task takes a day.